### ISSUE TITLE
Error Exporting Features from Recent Versions

### ISSUE THREAD

**Issue by philip-alldredge** [osate/osate2-core#549]
I recently updated my osate repository from develop. The updated code works fine until I try to export the features. I get an error from eclipse that ant has produced errors and placed them in a zip file. The root error is below

```
12. ERROR in Y:\Projects\osate2\repo\org.osate.xtext.aadl2.ui\src\org\osate\xtext\aadl2\ui\contentassist\AnnexAwareCompletionProposalComputer.java (at line 143)
	throw t;
	^^^^^^^^
Unhandled exception type Throwable
```

As a workaround I modified the code to catch Exception instead of Throwable and the issue was resolved. Suggestions?

**Comment by lwrage** [comment]
The try/catch is not needed so I'll remove it.

**Comment by lwrage** [comment]
fixed in 842ae257ec2dce6841118a62088d3da329900f36
